utils/date.ts

```diff
@@ -17,19 +24,27 @@ export const weeksBetween = (start: Date, end: Date) => {
   return raw.map(convert)
 }
 
+export const withinBounds = (start: Date, end: Date) => (e: ApiEvent) => {
+  const time = toDate(e.occurred_at)
+  return isAfter(time, start) && isBefore(time, end)
+}
+
 export const eventsBetween = (
   start: Date,
   end: Date,
   events: ApiEvent[]
-): ApiEvent[] =>
-  events.filter(e => {
-    const time = toDate(e.occurred_at)
-    return isAfter(time, start) && isBefore(time, end)
-  })
+): ApiEvent[] => events.filter(withinBounds(start, end))
 
 export const formatInTimeZone =
   (timeZone: string) => (date: Date, formatString: string) =>
     format(utcToZonedTime(date, timeZone), formatString, { timeZone })
 export const formatUTC = formatInTimeZone('UTC')
 export const formatEventDate = (d: Date) =>
   formatUTC(d, `y'-'MM'-'dd HH':'mm':'ss`)
+
+export const nextMondayFrom = (now: Date) => {
```

Comment:
`now` can be any time now right, since this code was factored out?

Reply:
Yeah, it can -- would you prefer a different parameter name?

Reply:
sort of, it just seemed like the code moved and the name just wasn't changed since that variable used to be `now` but isn't anymore. I'd say something like `time` or `from` but I don't think it matters that much. this is just a nit more than anything